hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/dao/GatewayServiceSpanDao.java

```diff
@@ -25,31 +21,29 @@
   private final GraphQlGrpcContextBuilder grpcContextBuilder;
   private final GatewayServiceSpanRequestBuilder requestBuilder;
   private final GatewayServiceSpanConverter spanConverter;
+  private final SpanLogEventFetcher spanLogEventFetcher;
 
   @Inject
   GatewayServiceSpanDao(
-      GraphQlServiceConfig serviceConfig,
-      CallCredentials credentials,
+      GatewayServiceFutureStubProvider gatewayServiceFutureStubProvider,
```

Comment:
Usually, you'd inject the `GatewayServiceFutureStub` rather than the provider itself